packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -36,6 +36,8 @@ const selectors = {
 	previewButton: ':is(button:text("Preview"), a:text("Preview"))',
 	publishButton: ( parentSelector: string ) =>
 		`${ parentSelector } button:text("Publish")[aria-disabled=false]`,
+	updateButton: 'button:text("Update")',
+	switchToDraftButton: 'button.editor-post-switch-to-draft',
```

Comment:
The button that handles saving/unpublishing (commonly referred to as the draft button) has different selectors based on the state of the post, hence this new selector to locate the button when the post is in an unpublishable state.